synapse/rest/client/v1/room.py

```diff
@@ -510,7 +523,9 @@ async def on_POST(self, request, room_id):
         # Prepend the insertion event to the start of the chunk
         events_to_create = [insertion_event] + events_to_create
 
-        inherited_depth = await self.inherit_depth_from_prev_ids(prev_events_from_query)
+        inherited_depth = await self._inherit_depth_from_prev_ids(
```

Comment:
Just renaming this with an underscore to align with the other internal functions